R/Rules-methods.R

```diff
@@ -2183,6 +2261,16 @@ setMethod("stopTrial",
       ## so can we stop?
       doStop <- probTarget >= stopping@prob
 
+      report_label <- if (isTRUE(stopping@report_label == "")) {
+        paste(
+          "P(", x@target[1], "\u2264", "Biomarker \u2264", x@target[2],
```

Comment:
same here: what is the `x` here? I do not see this object on the list of explicit arguments to this function.

Reply:
the content of the `paste()` can be in 1 line.